src/controller/db.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Controller, Get } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { DatabaseConfig } from '../config/interfaces';
+
+@Controller('db')
+export class DbController {
+  constructor(private readonly configServise: ConfigService) {}
```

Comment:
Do we need this controller?

Reply:
Oops, this file came by accident, it is not needed, I used it to test the functionality